CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.0] - 2024-08-27
+
+### Changed
+
+- Update Rusty Falcon to use the latest OpenAPI spec
+- Update example code to follow the latest OpenAPI spec changes
+- Fix links in the README
+
```

Comment:
As we have above the `Unreleased` section I would say we also may want to add a `Released` section for the code which has actually been released.